### DISCUSSION TITLE
zh/models/sam-2/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#14807] [Docs]
# zh/models/sam2/

探索 SAM2，Meta 的下一代 Segment Anything Model，它支持图像和视频中的实时提示分割，具有最先进的性能。了解其主要功能、数据集和使用方法。

https://docs.ultralytics.com/zh/models/sam-2/

**Comment by ALixuhui** [comment]
Is the reference address of sam2's paper incorrect?

**Reply by pderrenger** [reply]
@ALixuhui thank you for bringing this to our attention. If you believe there is an error in the reference address of SAM2's paper, please provide more details or the correct reference. We will review and update the documentation accordingly. Your feedback is valuable in ensuring our resources are accurate and helpful.

**Reply by glenn-jocher** [reply]
@ALixuhui should be fixed now!
